engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/pdf/util/HTML2Content.java

```diff
@@ -631,6 +633,7 @@ static void handleElement(Element ele, Map<Element, StyleProperties> cssStyles,
 				if ("".equals(text.getText())) // add default list type when tag <ul> attribute is empty.
 				{
 					text.setText("\u2022"); // the disc type
+					text.setTagType("Lbl"); // TODO HVB check if this is correct
```

Comment:
Is here a verification ongoing or done?

Reply:
Neither nor. I don't know if this is corect. I hope someone else who knows more about tagged PDF can tell us.